vendor/gx/ipfs/QmYVXrKrKHDC9FobgmcmshCDyWwdrfwfanNQN4oxJ9Fk3h/go-libp2p-peer/peer.go

```diff
@@ -169,8 +169,21 @@ func IDFromPublicKey(pk ic.PubKey) (ID, error) {
 	return ID(hash), nil
 }
 
-// OpenBazaar: temporary helper function to remain forward compatible with
-// inline keys
+// HashedIDFromPublicKey will always return the SHA256 hash of
+// the pubkey bytes. OpenBazaar: temporary helper to isolate the
+// hash-producing ID behavior.
+func HashedIDFromPublicKey(pk ic.PubKey) (ID, error) {
+	b, err := pk.Bytes()
+	if err != nil {
+		return "", err
+	}
+	hash, _ := mh.Sum(b, mh.SHA2_256, -1)
```

Comment:
This error shouldn't be swallowed

Reply:
Only thing I would prefer in these three patches to peer is for the comment to start with `// OpenBazaar:` so that we can find them later with a search.

Reply:
@cpacia The line includes `OpenBazaar:`. Is that sufficient here?

Reply:
Fixed the error handling.

Reply:
Probably sufficient. I would maybe also add it to the functions that were added but at least the change is noted.